SU2_CFD/src/output_paraview.cpp

```diff
@@ -909,6 +918,44 @@ void COutput::SetParaview_ASCII(CConfig *config, CGeometry *geometry, unsigned s
 
     }
     
+    if ((Kind_Solver == DISC_ADJ_FEM) && (config->GetFSI_Simulation())) {
```

Comment:
Should this also go in the Tecplot output?

Reply:
Yep, I am currently working on improved output for FSI, I will add this comment for a pull request to come (I thought this one was too large already, but I am working on some of these issues).